walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +272,153 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void optionalAttendeeUnavailable() {
```

Comment:
I was struggling with test case names when I originally wrote them because the names they had were super short--this convention makes a lot more sense. I tried to mimic this style but worry my new names might be too lengthy--please let me know if I can improve them.